Assignment3/src/com/csdisco/qa/Bicycle.java

```diff
@@ -42,12 +55,12 @@ void updateSpeed (){
 
         int inchesPerMin;
         float milesPerHour;
-        int tireCircumference = 80;
+        //int tireCircumference = 80;  // moved this up to be a field in class Bicycle since now the variable wheel class will use it also
         int tireRPM = 0;
         int inchesPerMile = 63360; // there are 63360 inches in a mile
```

Comment:
Fields like this are commonly made constants (class-global, private static final).